docs/examples/tutorial/string/to_unicode.py

```diff
@@ -0,0 +1,21 @@
+from cython.cimports.cpython.version import PY_MAJOR_VERSION
+
+# @cython.cfunc  # https://github.com/cython/cython/issues/2529
+def _text(s) -> unicode:
```

Comment:
We should probably use `cython.unicode` here as well (and below). Otherwise, it wouldn't work in plain Py3.

Reply:
sure

Reply:
What's interesting, without changing `unicode` to `str` it won't work even with `cython.unicode` because of the `if type(s) is unicode`

It won't run in python either way

One option is to replace `unicode` with `str`, nobody is using unicode anyways nowadays, the code that was written a long time ago will still work and people who's supporting it will know what to do with it, I think, on the contrary the new users will only be confused

Second options is to make a remark about the fact that this will ru only in python 2

And probably the best way is to do both, but leave the unicode version be, while having a str version as the first thing people see in the docs

Reply:
I guess we can also use `type(s) == unicode` and add the `__eq__` to `typedef` in the Shadow.py

Reply:
Maybe use `language_level=3` and `str`?